lib/client.ml

```diff
@@ -210,15 +210,15 @@ let write_one t from buffer =
 
 let write t from buffers =
   if t.disconnected
-  then return (Error `Disconnected)
+  then Lwt.return (Error `Disconnected)
   else begin
     let rec loop from = function
-      | [] -> return (Ok ())
+      | [] -> Lwt.return (Ok ())
```

Comment:
I believe there is `Lwt.return_ok` and `Lwt.return_error`

Reply:
Indeed, and it's in Lwt 2.7.1, which is in xs-opam: http://ocsigen.org/lwt/2.7.1/api/Lwt :smile: 
Changed all occurrences to these functions.